contracts/test/utils/TestMulticall.sol

```diff
@@ -22,6 +22,11 @@ contract TestMulticall is Multicall {
         return nonce_;
     }
 
+    function function3() external pure {
+        // solhint-disable-next-line reason-string
+        revert();
```

Comment:
maybe add a revert string due to the warning?

Reply:
yup, will add